chsdi/templates/htmlpopup/cadastralwebmap.mako

```diff
@@ -6,7 +6,7 @@
 <%def name="table_body(c, lang)">
     <%
       from chsdi.models.vector import getScale
-      from chsdi.lib.validation.features import IdentifyServiceValidation
+      from chsdi.lib.validation.identify import IdentifyServiceValidation
```

Comment:
Did you test this change? There are differences between the two IdentifyServiceValidation classes. See https://www.diffchecker.com/q6hgmxnq (the one you removed here is on the right)